internal/commands/validate.go

```diff
@@ -4,6 +4,9 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/docker/app/internal/compose"
```

Comment:
import order looks wrong

Reply:
? I ran goimports

Reply:
ok so any blank line just prevent goimports to work the way I expected it to. Nice to know :roll_eyes: